ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -323,22 +324,22 @@ describe('EthOverview', () => {
       };
       const mockedStore = configureMockStore([thunk])(mockedFantomStore);
 
-      const { queryByTestId, queryByText } = renderWithProvider(
+      const { queryByTestId } = renderWithProvider(
         <EthOverview />,
         mockedStore,
       );
       const bridgeButton = queryByTestId(ETH_OVERVIEW_BRIDGE);
       expect(bridgeButton).toBeInTheDocument();
       expect(bridgeButton).toBeDisabled();
-      expect(queryByText('Bridge').parentElement).toHaveAttribute(
+      expect(bridgeButton.parentElement).toHaveAttribute(
         'data-original-title',
         'Unavailable on this network',
       );
     });
 
     it('should always show the Portfolio button', () => {
       const { queryByTestId } = renderWithProvider(<EthOverview />, store);
-      const portfolioButton = queryByTestId(ETH_OVERVIEW_RECEIVE);
+      const portfolioButton = queryByTestId('portfolio-link');
```

Comment:
Pretty sure this test was incorrectly looking for the receive data test id instead of the portfolio link